Formula/networkit.rb

```diff
@@ -4,10 +4,9 @@ class Networkit < Formula
   desc "Performance toolkit for large-scale network analysis"
   homepage "https://networkit.github.io"
   url "https://github.com/networkit/networkit.git",
-      tag:      "7.1",
-      revision: "4c6dcc4367b51005a34221242048609c357ffbd6"
+      tag:      "8.0",
+      revision: "bdbb926e7d3102c32a3fc54d4cfa769bfac822f4"
```

Comment:
Is there really any reason to keep pulling from git instead of using the tarball releases hosted on github?

Reply:
I don't have context for why it is using the tab and revision. Maybe the artifact issue? 🤷‍♂

Reply:
Can you try switching?

Reply:
test it out in my local right now.

Reply:
this is the reason why it was using `git checkout`.
```
CMake Error at CMakeLists.txt:210 (message):
  Missing TLX library in extlibs/tlx Please run `git submodule update --init`
  to fetch the submodule.
```

Reply:
we do have tlx formula, so it should be good.

Reply:
works for me and tested out. 🎉